testing on profile

p-wong1124 opened the pull request:
Here I put the profile testing to test on the behavior of this

for testing on the data read&write 
you can just run on the profile.html :P 

and also please change the firebase setting inside the js/site.js to test with your firebase database.
PS: I have changed the site.js to our common firebase server :)
Have fun

Commit:
Mounted to my firebase & testing on profile

Here I put the profile testing to test on the behavior of this
have fun :P

Commit:
should put the file to another branch to prevent affect on others

Commit:
for returning the previous two version

wai2016 commented on app/js/site.js:
```diff
@@ -29,12 +29,14 @@ function getRandomIntInclusive(min, max) {
 function initalizeFirebase() {
 	
   // Initialize Firebase
```
We have opened a new google account for all of us (comp3111helloworld) so that we can access the firebase without using different google ac. Using the new firebase config is better.

p-wong1124 replied:
I agree on you.
Will change to common firebase server instead, when pull request to this master :)

RaymondWongCH commented:
We can add the profile.html now and modify it later. Agree on using the new firebase.

Commit:
rename Detail page and update with new Firebase code (#1)

rename Detail page and update with new Firebase code

Commit:
Update site.js

Using commonly used firebase server

Commit:
Update site.js

Commit:
temperately removed detailpage to merge from common repo

Commit:
Merge remote-tracking branch 'refs/remotes/ust-helloworld/master'

# Conflicts:
#	app/js/site.js

Commit:
Putting back *Detail*.html

Commit:
minor amendment on site.js

wai2016 commented on app/memberDetailTemp.html:
```diff
@@ -0,0 +1,114 @@
+   
+   </div>
+   
+      <!-- Angular and Angular-route libraries (v1.5.7) -->
```
I think the libraries can put inside head

p-wong1124 replied:
ok ar
as just implemented it quickly lol

wai2016 commented on app/memberDetailTemp.html:
```diff
@@ -0,0 +1,114 @@
+	initalizeFirebase();
+	database = firebase.database();
+	
+	$scope.FUID = "001";
```
I think uid and name should be the one given by fb auth.

p-wong1124 replied:
because just use it as the autofill, changing it to none is better or changing it to login info is better :)

wai2016 replied:
I see

wai2016 commented on app/teamDetailTemp.html:
```diff
@@ -0,0 +1,124 @@
+	database = firebase.database();
+	
+	$scope.FName = "teamA";
+	$scope.FDescription = "Need C++";
```
Should the team leader create description options for members to select? What I mean is that all team leaders will create a list for members to choose when member editing their profile. As a result, team leader can search members more quicker because members must have the same description as team description.

p-wong1124 replied:
I don't quite understand what you mean by "all team leaders will create a list for members to choose when member editing their profile"...
Can you explain more ;) ??

wai2016 replied:
`$scope.FDescription = "Need C++";`
Then member should have an option called C++ in their profile.

wai2016 commented on app/teamDetailTemp.html:
```diff
@@ -0,0 +1,124 @@
+		FSize = $scope.FSize;
+		if ( FName !== '' && FActivity !=='') {
+			
+			var refPath = FActivity + "/team/"+FName;
```
We use FName to locate the firebase here, right? So FName must be unique and mandatory. Or we can make a TeamID for all teams.

p-wong1124 replied:
FName here is just mean the TeamID only.

We can discuss should we use TeamID for identifying teams, and they can use different display name :)

wai2016 replied:
Oh, team name in fact should be unique. XD
Then the problem is solved.

wai2016 reviewed (commented):
Good, admin profile left. Do you want to do that or let me do that?
And I have reviewed your codes with some recommendations. =]

Commit:
Using 2 white spaces instead of 1 tab.
And also put libraries to the <head>

Commit:
Merge remote-tracking branch 'refs/remotes/ust-helloworld/master'

Commit:
Added a activity Detail

Using template to create a simple activity detail page
And try to add somethings like to display message.
This template use update to save on the database :)

wai2016 reviewed (commented):
Merge remote-tracking branch 'refs/remotes/ust-helloworld/master'
How to do this? 
I don't know. T.T

p-wong1124 commented:
Seem using pull request to pull ust-helloworld to my repo :P
